ServerNotifications.Web/Domain/Notifier.cs

```diff
@@ -1,26 +1,40 @@
 ﻿using ServerNotifications.Web.Domain.Twilio;
 using ServerNotifications.Web.Models.Repository;
+using System;
+using System.Collections.Generic;
+using Twilio.Clients;
 using WebGrease.Css.Extensions;
 
 namespace ServerNotifications.Web.Domain
 {
     public class Notifier
     {
-        private const string ImageUrl = "http://howtodocs.s3.amazonaws.com/new-relic-monitor.png";
+        private Uri ImageUrl = new Uri("http://howtodocs.s3.amazonaws.com/new-relic-monitor.png");
 
         private readonly IAdministratorsRepository _administratorsRepository;
         private readonly IRestClient _restClient;
 
-        public Notifier(IAdministratorsRepository repository, IRestClient restClient)
+        public Notifier(IAdministratorsRepository repository)
         {
             _administratorsRepository = repository;
-            _restClient = restClient;
+            _restClient = new RestClient();
         }
-        
-        public void SendMessages(string message)
+
+        public Notifier(IAdministratorsRepository repository, ITwilioRestClient client)
+        {
+            _administratorsRepository = repository;
+            _restClient = new RestClient(client);
+        }
+
+        public async void SendMessages(string message)
```

Comment:
The type signature for this one should be:

```csharp
public async Task SendMessages(string message);
```

What about changing the method's name to `SendMessagesAsync`? It makes explicit that you're dealing with an `async` method.